examples/tutorial_2_customproto.md

```diff
@@ -54,23 +60,23 @@ proc main() {.async, gcsafe.} =
     testProto = TestProto.new()
     switch1 = newStandardSwitch(rng=rng)
     switch2 = newStandardSwitch(rng=rng)
-  
+
   switch1.mount(testProto)
-  
+
   await switch1.start()
   await switch2.start()
-    
   let conn = await switch2.dial(switch1.peerInfo.peerId, switch1.peerInfo.addrs, TestCodec)
 
+  # conn is now a fully setup connection, we talk directly to the switch1 custom protocol handler
   await testProto.hello(conn)
 
-  # We must close the connection ourselves when we're done with it
+  # We must close the connections ourselves when we're done with it
```

Comment:
This sentence is talking specifically about `conn`, the connection we got from dial.

This is actually a stream and not a connection: we need to close the stream, and libp2p will handle to close the connection

Reply:
Gotcha, thanks. Should the comment on line 70 be:
```
# conn is now a fully setup stream, which we use to talk directly to the switch1 custom protocol handler
```
?

Reply:
Or maybe we should just leave it with no comment 😄

Reply:
Yeah, we can rename it stream, and explain the flow better in part 1
I can do it if you want

Reply:
I'd definitely appreciate it, for my own learning 😄 🙏 

Note, i've just force pushed (with lease), in case you've pulled the branch already.

Reply:
Did something, LMK what you think :)